MorphicServer/HttpExtensions.cs

```diff
@@ -47,6 +47,12 @@ public static async Task WriteJson<T>(this HttpResponse response, T obj, Cancell
                 {
                     Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping
                 };
+                // For BadRequestResponse omit empty 'details'
```

Comment:
Do you think we should do this for all objects? Or do you want to see empty fields always (even with errors)?

Reply:
I don't think we need this.  The strict typed clients will behave the same regardless of whether the field is missing or null.  Keeps the code cleaner if we don't have type checked exceptions to the rule

Reply:
I guess I should have made the above a review comment

Reply:
Do I click the "Resolve conversation" button? Or do you?